src/averageAges.js

```diff
@@ -37,7 +38,12 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
```

Comment:
The condition in the filter function should check if the person's sex is 'f' and there is at least one person who mentions her as a mother. The current condition is incorrect.